src/clib/gaussj_g.cpp

```diff
@@ -0,0 +1,76 @@
+#include <numeric>
```

Comment:
```suggestion
#include <cmath>
```

I don't think you are actually using anything from `<numeric>`.

You forgot to include `<cmath>` (so that we can take absolute values)